NOTICE.md

```diff
@@ -0,0 +1,59 @@
+# Notices for Eclipse Project for WebSocket
+
+This content is produced and maintained by the Eclipse Project for WebSocket
+project.
+
+* Project home: https://projects.eclipse.org/projects/ee4j.websocket
+
+## Trademarks
+
+Eclipse Project for WebSocket is a trademark of the Eclipse Foundation.
+
+## Copyright
+
+All content is the property of the respective authors or their employers. For
+more information regarding authorship of content, please consult the listed
+source code repository logs.
+
+## Declared Project Licenses
+
+This program and the accompanying materials are made available under the terms
+of the Eclipse Public License v. 2.0 which is available at
+http://www.eclipse.org/legal/epl-2.0. This Source Code may also be made
+available under the following Secondary Licenses when the conditions for such
+availability set forth in the Eclipse Public License v. 2.0 are satisfied: GNU
+General Public License, version 2 with the GNU Classpath Exception which is
+available at https://www.gnu.org/software/classpath/license.html.
+
+SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+
+## Source Code
+
+The project maintains the following source code repositories:
+
+* https://github.com/eclipse-ee4j/websocket-api
+
+## Third-party Content
```

Comment:
I can't see where these third-party libraries are used. As far as I am aware, the WebSocket JARs do not have any external dependencies.

Reply:
That's a darn good question.
That text comes as-is from the generated "legal documents" at https://www.eclipse.org/projects/tools/documentation.php?id=ee4j.websocket

Of the 3 references I see ...

* atinject (Package javax.inject) (1.0)
* Google Guava (18.0)
* javax.validation:validation-api:jar:1.1.0.Final (1.1.0)

This project (eclipse-ee4j/websocket-api) does not use and has no reference to those dependencies.

* Not for direct dependencies in pom
* Not included in distributed artifacts
* Nor for can-use undistributed references
* Not for build / testing reasons

The only 3rd party references we have are to the Servlet API, and even then it's solely present as javadoc comments.

https://github.com/eclipse-ee4j/websocket-api/blob/bb4326680f9ebddf38fdc92cbe0156f1525547d0/api/server/src/main/java/javax/websocket/server/ServerContainer.java#L26-L31

I don't like changing "legal documents" without letting the Eclipse Foundation (and in this case the ee4j-pmc) know first.

So, @m0mus or @waynebeaton can we get approval to replace the entire `## Third-party content` section with ...

```
## Third-party Content

This project leverages no third party content.
```

in this project?

Reply:
Looks like @bravehorsie has confirmed that those 3 dependencies belong to tyrus.
eclipse-ee4j/tyrus#645

I've filed an IP/CQ for the reassignment of those dependencies with Eclipse IP / Legal.
Once they have been updated, I'll regenerate and update this existing PR.

Reply:
I've bumped the ipzilla issue, hopefully this gets addressed soon.